styles/lemmy/catppuccin.user.css

```
@@ -0,0 +1,234 @@
+/* ==UserStyle==
+@name lemmy Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/lemmy
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/lemmy
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/lemmy/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Alemmy
+@description Soothing pastel theme for lemmy
```

(see previous suggestion)
```suggestion
@name Lemmy Catppuccin
@namespace github.com/catppuccin/userstyles/styles/lemmy
@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/lemmy
@version 0.0.1
@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/lemmy/catppuccin.user.css
@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Alemmy
@description Soothing pastel theme for Lemmy
```